server/test/com/cloud/user/AccountManagerImplTest.java

```diff
@@ -185,6 +186,8 @@
     @Mock
     GlobalLoadBalancerRuleDao _gslbRuleDao;
     @Mock
+    SSHKeyPairDao _sshKeyPairDao;
```

Comment:
@borisroman the unit tests will fail without this change.
anyway, I will add some codes in the test file.